### PR TITLE
Ability to configure testable email recipient domains

### PR DESCRIPTION
**Pull request by jerbri**

### CODE REVIEW

**Comment by mattgwagner** [comment]
```
"C:\projects\certipay-common\CertiPay.Common.Notifications\CertiPay.Common.Notifications.csproj" (default target) (5) ->
(CoreCompile target) -> 
  Notifications\IEmailService.cs(49,51): error CS0103: The name 'ConfigurationManager' does not exist in the current context [C:\projects\certipay-common\CertiPay.Common.Notifications\CertiPay.Common.Notifications.csproj]
 
    144 Warning(s)
    1 Error(s)
 
Time Elapsed 00:01:21.58
```

File: CertiPay.Common.Notifications/Notifications/IEmailService.cs [hunk 1]
```diff
@@ -36,6 +37,7 @@ public class EmailService : IEmailService
         /// <summary>
         /// A list of domains that we will allow emails to go to from outside of the production environment
         /// </summary>
+        /// <remarks>Override with comma-separate appSetting 'EmailService.AllowedTestingDomains'</remarks>
         public IEnumerable<String> AllowedTestingDomains { get; set; }
```
Comment (mattgwagner) [inline CertiPay.Common.Notifications/Notifications/IEmailService.cs]: Would it be simpler if we just make `AllowedTestingDomains` a static property that the calling application can set? That way, the Configuration remains an Application-concern?

Comment (jerbri) [inline CertiPay.Common.Notifications/Notifications/IEmailService.cs]: Agreed. That would be better.

**Comment by mattgwagner** [comment]
@jerbri did you want to make the change or want me to just fix it on master?

**Comment by mattgwagner** [comment]
Completed via ed7f4bb6e54b08147b56df84d86bc4b0cba04eca

**Pull request closed without merging**